index.js

```diff
@@ -182,12 +182,15 @@ const create = (win, options) => {
 		// TODO: https://github.com/electron/electron/issues/5869
 		menuTemplate = removeUnusedMenuItems(menuTemplate);
 
-		// Apply custom labels for default menu items
-		if (options.labels) {
-			for (const menuItem of menuTemplate) {
-				if (options.labels[menuItem.id]) {
-					menuItem.label = options.labels[menuItem.id];
-				}
+		for (const menuItem of menuTemplate) {
+			// Apply custom labels for default menu items
+			if (options.labels && options.labels[menuItem.id]) {
+				menuItem.label = options.labels[menuItem.id];
+			}
+
+			// Replace placeholders in menu item labels
+			if (typeof menuItem.label === 'string') {
+				menuItem.label = menuItem.label.replace('{selection}', cliTruncate(props.selectionText.trim(), 25));
```

Comment:
I don't think `.selectionText` is guaranteed to exist.

Reply:
Hmm, good point. I will have to guard against that.

Sorry for all the back and forth; I hadn't anticipated having to change so many things in addition to the actual implementation of the menu item! 😊

Reply:
Actually it seems it does always exist, but better to  guard against it anyway.